db/sessions/model.js

```diff
@@ -11,21 +11,57 @@ const Schema = mongoose.Schema;
 let sessionsSchema = new Schema({
   title: String,
   description: String,
-  startDate: Date,
-  endDate: Date,
   recommendedFor: Array,
-  category: Array,
-  medium: String,
-  image: String,
+  categories: Array,
+  medium: Array,
+  image: Array,
   userId: {
     type: String,
     index: true
   },
+  mentorName: String,
   sessionReschedule: Number,
   status : {
     type: String,
-    index: true
-  }
+    index: true,
+    default: "published"
+  },
+  deleted: {
+    type: Boolean,
+    default: false
+  },
+  timeZone:String,
+  startDate: String,
+  endDate: String,
+  startDateUtc: String,
+  endDateUtc: String,
+  link: String,
+  menteePassword: String,
+  mentorPassword: String ,
+  startedAt: String,
+  shareLink: String,
+  bigBlueButtonMeetingInfo: Object,
```

Comment:
@aman-tunerlabs we are not making use of bigBlueButtonMeetingInfo key currently, so it can be removed ?

Reply:
@aman-g202  bigBlueButtonMeetingInfo is required for data pipelines team.

Reply:
But we are not saving anything on this key...?